etc/adminhtml/system.xml

```diff
@@ -39,11 +39,11 @@
                 </field>
 
                 <group id="omise_cc" translate="label" sortOrder="80" showInDefault="1" showInWebsite="1" showInStore="1">
-                    <label>Enable Credit Card Solution</label>
+                    <label>Enable Credit Card Payments</label>
                     <comment>Powerful payment features that allows you to easily and securely accept credit/debit card payments on your store.</comment>
                     <field id="active" translate="label comment" type="select" sortOrder="90" showInDefault="1" showInWebsite="1" showInStore="1">
-                        <label>Enable/Disable</label>
-                        <comment>Enable Omise Credit Card Solution.</comment>
+                        <label>Enabled?</label>
+                        <comment>Enable Omise Credit Card Payments.</comment>
```

Comment:
Do we need 'Omise'? This will appear within the settings for the Omise plugin anyway

Reply:
'Omise' doesn't appear in most of the other methods below

Reply:
I removed it.